cmd/mattermost/commands/command.go

```diff
@@ -268,3 +290,97 @@ func deleteCommandCmdF(command *cobra.Command, args []string) error {
 	CommandPrettyPrintln("Deleted command '" + slashCommand.Id + "' (" + slashCommand.DisplayName + ")")
 	return nil
 }
+
+func modifyCommandCmdF(command *cobra.Command, args []string) error {
+	a, err := InitDBCommandContextCobra(command)
+	if err != nil {
+		return err
+	}
+	defer a.Shutdown()
+
+	oldCommand := getCommandFromCommandArg(a, args[0])
+	if oldCommand == nil {
+		command.SilenceUsage = true
+		return errors.New("Unable to find command '" + args[0] + "'")
+	}
+	modifiedCommand := oldCommand
+
+	// get creator user
+	creator, _ := command.Flags().GetString("creator")
+	if creator != "" {
+		user := getUserFromUserArg(a, creator)
+		if user == nil {
+			return errors.New("unable to find user '" + creator + "'")
+		}
+
+		// check if creator has permission to create slash commands
+		if !a.HasPermissionToTeam(user.Id, modifiedCommand.TeamId, model.PERMISSION_MANAGE_SLASH_COMMANDS) {
```

Comment:
I'm ok with this for this PR because is already checked in the `CreateCommandCmd` but we normally avoid permissions checks and things like that in the CLI command, because if you have CLI command we understand that you have full privileges. Anyway, as I said, this should be discussed and fixed in another PR, for now I prefer the existing consistency between both commands.